Dockerfile

```diff
@@ -34,15 +38,12 @@ LABEL description="RADAR-CNS Backend- HDFS Sink Connector"
 
 ENV CONNECT_PLUGIN_PATH /usr/share/java/kafka-connect/plugins
 
-COPY --from=builder /code/build/libs/*.jar /usr/share/java/kafka-connect/plugins/hdfs-sink-connector-radar/
-COPY --from=builder /code/build/third-party/*.jar /usr/share/java/kafka-connect/plugins/hdfs-sink-connector-radar/
-
 # To isolate the classpath from the plugin path as recommended
-COPY --from=builder /code/build/libs/*.jar /etc/kafka-connect/hdfs-sink-connector-radar/
-
+COPY --from=builder /code/build/third-party/*.jar ${CONNECT_PLUGIN_PATH}/radar-hdfs-sink-connector/
+COPY --from=builder /code/build/libs/*.jar ${CONNECT_PLUGIN_PATH}/radar-hdfs-sink-connector/
```

Comment:
The jar should also be copied at the classpath (something like `/etc/kafka-connect/radar-hdfs-sink-connector`) as its good to isolate the `CLASSPATH` and `PLUGINPATH`. This would also help in the future if we add these custom classes to a radar-commons module. Also it helps loading the third-party libs only with the plugin. Alternatively, changing the `CLASSPATH` in `src/main/docker/launch` equal to `PLUGINPATH` should work too.